sklearn/feature_selection/_base.py

```diff
@@ -214,10 +223,14 @@ def _get_feature_importances(estimator, getter, transform_func=None, norm_order=
                 )
         else:
             getter = attrgetter(getter)
-    elif not callable(getter):
```

Comment:
This check has been removed because in https://github.com/scikit-learn/scikit-learn/blob/4ee3fdd85c8d78da24c166a4127e8bc7bc8dd469/sklearn/feature_selection/_rfe.py#L250 we already check whether instance_getter is either a string or a callable. Furthermore, I have added a test case below to verify that a value error is raised when `RFE.fit` is called with an importance getter is not a callable or a string.